yagb-core/src/emulator/ppu/ppu-cgb.ts

```diff
@@ -345,6 +363,42 @@ export class PpuCgb extends PpuBase {
         this.updatePalette(this.paletteOB1, value);
     };
 
+    private bgpiRead: ReadHandler = (_) =>
+        this.bgpi;
+    private bgpiWrite: WriteHandler = (_, value) => {
+        this.bgpi = value & 0xff;
```

Comment:
The `&` is not strictly necessary, but from the spec it seems that bit 6 is not driven and probably should be always high (maybe cross-check with the SameBoy source).